src/index.ts

```diff
@@ -5,5 +5,6 @@
  */
 export { default as mainMw } from './main-mw.ts';
 export { load as init } from './server-settings.ts';
-
 import './db-types.ts';
+export * from './types.ts';
```

Comment:
Ideally we only export `api-types.ts`. I know that doesn't solve your immediate problem, but only api-types.ts is the 'server contract' that we publish.

If you're open to it, you could make a small json-schema for 'ServerStats`, (you can re-generate api-types using a script in `bin/`)

`types.ts` is internal and if we export it, changing those types means we're breaking backwards compatibility.

Reply:
Oh LOL! I based on it named as `src/types.ts` I thought it was authoritative but just missing
yeah that sounds good